internal/controller/common.go

```diff
@@ -270,7 +270,7 @@ func (c *common) OnDelete(addon *internal.CommonAddon) error {
 func (c *common) loadRepositories(repos []v1alpha1.SpecRepository) *repository.Collection {
 	repositories := repository.NewRepositoryCollection()
 	for _, specRepository := range repos {
-		c.log.Infof("- create addons for %q repository", specRepository.URL)
```

Comment:
why do you remove URL from logs? sensitive data is usually as URL parameters so maybe we can just remove this part at the beginning of loop?
```go
repositoryURL := strings.Split(url, "?")[0]
```